christfellowshipchurch/web-app
# ESLint Error Roundup

Wipe out remaining ESLint errors from console and a few bonus ones.

### To test:
- Play around with the app and make sure nothing looks out of place or behaves oddly.
- Locally, there should be no more ESLint warnings
